test/tests_for_sdkmr/ee2_load_test.py

```diff
@@ -85,7 +86,9 @@ def test_init_job_stress(self):
         method_1 = "app_1.a_method"
         method_2 = "app_1.b_method"
         job_params_1 = get_sample_job_params(method=method_1)
+        job_params_1["job_reqs"] = JobRequirements(1, 1, 1, "njs")
```

Comment:
This is another pattern that will repeat throughout the tests. In this case `get_sample_job_params` creates a perfectly fine set of job parameters if they're supplied to the `EE2RunJob` API. However, this test bypasses the API and submits the parameters to the internals directly via `_init_job_rec`. As such, the parameters need to be patched up to match the processing they normally would've undergone in `EE2RunJob` prior to hitting `_init_job_rec` to avoid an error.